### PR TITLE
feat(elasticsearch): Upgrade/elastic

### PR DESCRIPTION
**Pull request by rauerhans**
## Summary

- Upgrade helm chart  from `2.4.0` to `2.8.0`
- Upgrade ECK operator from `2.4.0` to `2.8.0`
- Upgrade supported ELK app versions from `7.13.2` to `7.17.0`
  - newest version is actually `8.8.0`, however there's an upgrade path that foresees a stepwise approach that requires an upgrade to specifically `7.17.0` first. We should let it sit at `7.17.0` for a while and then upgrade from there at a later date, otherwise we might break people's running ELK clusters. See also: https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html
- vendoring operator image now
- not vendoring the ELK stack images, there are too many of them

## Checklist

- [ ]  No images hosted from dockerhub
- [ ]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [ ]  all databases should have dashboards
    - [ ]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [ ]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [ ]  Are scaling runbooks present
    - [ ]  all databases **must** have scaling runbooks
    - [ ]  you can use the charts in `pluralsh/module-library` to accelerate this
- [ ]  do you need to add config overlays?
    - [ ]  inputing secrets
    - [ ]  configuring autoscaling
- [ ]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable
- [ ]  All major clouds must be supported
    - [ ]  Azure
    - [ ]  AWS
    - [ ]  GCP

### CODE REVIEW

File: elasticsearch/helm/elasticsearch/values.yaml [hunk 1]
```diff
@@ -39,18 +43,21 @@ elasticsearch:
       memory: 4Gi
       cpu: 2
   storage: 50Gi
-  version: '7.13.2'
+  version: '7.17.0'
```
Comment (DavidSpek) [inline elasticsearch/helm/elasticsearch/values.yaml:46]: Why is this a string while the others aren't?

**Review by DavidSpek — CHANGES_REQUESTED** [review]
Can you setup the CRDs so that they use the vendored images? All `7.17.0` images should be available now from our registry.

**Review by michaeljguarino — APPROVED** [review]

**Review by michaeljguarino — APPROVED** [review]

**Comment by rauerhans** [comment]
> Can you setup the CRDs so that they use the vendored images? All `7.17.0` images should be available now from our registry. this is done

**Review by DavidSpek — APPROVED** [review]
LGTM

**Pull request merged**